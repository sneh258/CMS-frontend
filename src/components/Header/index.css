.header-comp {
  /* border: 1px solid red; */
  padding-left: 35px;
  font-family: 'Montserrat', sans-serif;
  font-weight: 700;
  height: 37px;
  padding-left: 10px;
}