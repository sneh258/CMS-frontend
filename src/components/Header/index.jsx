import PropTypes from 'prop-types';
import React from 'react';
import './index.css';
import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
import { faUser } from '@fortawesome/free-solid-svg-icons';

export default function Header({ text }) {
  return (
    <div className="header-comp">
      <p>{text}</p>
      <div className='logout'><FontAwesomeIcon icon={ faUser}/></div>
    </div>
  );
}
Header.propTypes = {
  text: PropTypes.string.isRequired,
};
