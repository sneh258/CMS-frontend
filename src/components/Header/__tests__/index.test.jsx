import React from 'react';
import Header from '..';
import { render } from '@testing-library/react';

const mockNavigate = jest.fn();
jest.mock('react-router-dom', () => ({
  useNavigate: () => mockNavigate
}));

describe('Header', () => {
  it('should render correctly', () => {
    const { asFragment } = render(<Header text="company" />);
    expect(asFragment()).toMatchSnapshot();
  });
});
