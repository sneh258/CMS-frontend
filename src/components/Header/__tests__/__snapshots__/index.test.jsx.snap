// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should render correctly 1`] = `
<DocumentFragment>
  <div
    class="header-comp"
  >
    <p>
      company
    </p>
  </div>
</DocumentFragment>
`;
